### PR TITLE
Deployment 2

### PR DESCRIPTION
**Pull request by tlake**

### CODE REVIEW

File: iterative-walkthrough/deployment-1/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Best practice would be to gitignore this file to not share secrets.
```
Comment (diemonster) [inline iterative-walkthrough/deployment-1/terraform.tfvars]: Remove below....

File: iterative-walkthrough/deployment-1/guestbook/main.go [hunk 2]
```diff
@@ -9,16 +9,19 @@ import (
 var entries []string
 
 func listEntries(c *fireball.Context) (fireball.Response, error) {
+    log.Println("list")
```
Comment (diemonster) [inline iterative-walkthrough/deployment-1/guestbook/main.go]: debug statements meant to be here still?

**Review by diemonster — CHANGES_REQUESTED** [review]

File: iterative-walkthrough/deployment-1/terraform.tfvars (hunk 1 above)
Comment (tlake) [reply iterative-walkthrough/deployment-1/terraform.tfvars]: 😮 ✅

File: iterative-walkthrough/deployment-1/guestbook/main.go (hunk 2 above)
Comment (tlake) [reply iterative-walkthrough/deployment-1/guestbook/main.go]: ✅

**Comment by diemonster** [comment]
👍 future work will be to refactor/generalize base guestbook to be the same as in Deployment #3 example

**Review by diemonster — APPROVED** [review]

**Pull request merged**